Dockerfile

```diff
@@ -29,6 +30,7 @@ RUN apt-get update && apt-get dist-upgrade -y && apt-get -y install \
 
 # Build time variables
 ENV LSMB_VERSION 1.5.8
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
I think this should be removed entirely as an ENV setting, and used at the start of each RUN line where needed.